src/objectTranslations/Contact-ca.objectTranslation

```diff
@@ -3473,36 +3473,64 @@
         <name>PreferredPhone__c</name>
         <picklistValues>
             <masterLabel>Home</masterLabel>
-            <translation>Llar</translation>
+            <translation>Home Phone</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Mobile</masterLabel>
-            <translation>Mòbil</translation>
+            <translation>Mobile Phone</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Other</masterLabel>
-            <translation>Altres</translation>
+            <translation>Other Phone</translation>
         </picklistValues>
         <picklistValues>
             <masterLabel>Work</masterLabel>
-            <translation>Feina</translation>
+            <translation>Telèfon de la feina</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Home Phone</masterLabel>
+            <translation>Home Phone</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Mobile Phone</masterLabel>
+            <translation>Mobile Phone</translation>
+        </picklistValues>
+        <picklistValues>
+            <masterLabel>Other Phone</masterLabel>
+            <translation>Other Phone</translation>
         </picklistValues>
+        <picklistValues>
+            <masterLabel>Work Phone</masterLabel>
+            <translation>Telèfon de la feina</translation>
+        </picklistValues> 
     </fields>
     <fields>
         <help>Quin correu electrònic s&apos;hauria d&apos;utilitzar per a la majoria de comunicacions que incloguin aquest contacte?</help>
         <label>Correu preferit</label>
         <name>Preferred_Email__c</name>
         <picklistValues>
             <masterLabel>Alternate</masterLabel>
-            <translation>Alternatiu</translation>
+            <translation>orreu electrònic alternatiu</translation>
```

Comment:
Missing a C at the beginning, I think...

Reply:
Good catch! Updated it.